core/src/main/java/org/gbif/pipelines/core/parsers/legacy/CountryMaps.java

```diff
@@ -0,0 +1,89 @@
+package org.gbif.pipelines.core.parsers.legacy;
+
+import org.gbif.api.vocabulary.Country;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.Map;
+import java.util.Set;
+
+import com.google.common.collect.Maps;
+import com.google.common.collect.Sets;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Maps of countries that are commonly confused, or are considered equivalent.
+ */
+public class CountryMaps {
+
+  private static final Logger LOG = LoggerFactory.getLogger(CountryMaps.class);
+
+  private static final String CONFUSED_COUNTRY_FILE = "confused-country-pairs.txt";
+
+  private CountryMaps() {}
+
+  /*
+   Some countries are commonly mislabeled and are close to correct, so we want to accommodate them, e.g. Northern
+   Ireland mislabeled as Ireland (should be GB). Add comma separated pairs of acceptable country swaps in the
+   confused_country_pairs.txt file. Entries will be made in both directions (e.g. IE->GB, GB->IE). Multiple entries per
+   country are allowed (e.g. AB,CD and AB,EF).
+   This *overrides* the provided country, and includes an issue.
+  */
+  private static final Map<Country, Set<Country>> CONFUSED_COUNTRIES = Maps.newHashMap();
+  // And this is the same, but without the issue — we aren't exactly following ISO, but we accept it.
+  private static final Map<Country, Set<Country>> EQUIVALENT_COUNTRIES = Maps.newHashMap();
+
+  static {
+
+    InputStream in = CountryMaps.class.getClassLoader().getResourceAsStream(CONFUSED_COUNTRY_FILE);
```

Comment:
move this into try (InputStream in = CountryMaps.class.getClassLoader().getResourceAsStream(CONFUSED_COUNTRY_FILE);) {
}
you can also ignore the all the closing logic